acceptance-tests/src/main/java/io/blueocean/ath/model/Pipeline.java

```diff
@@ -1,81 +1,44 @@
 package io.blueocean.ath.model;
 
-import com.google.common.base.Joiner;
-import com.google.common.net.UrlEscapers;
-import com.google.inject.Inject;
-import io.blueocean.ath.BaseUrl;
-import io.blueocean.ath.factory.ActivityPageFactory;
-import io.blueocean.ath.factory.RunDetailsPipelinePageFactory;
-import io.blueocean.ath.pages.blue.ActivityPage;
-import io.blueocean.ath.pages.blue.RunDetailsPipelinePage;
-
-public abstract class Pipeline {
-    private Folder folder;
-    private String name;
-
-    @BaseUrl @Inject
-    String baseUrl;
-
-    @Inject
-    ActivityPageFactory activityPageFactory;
 
-    @Inject
-    RunDetailsPipelinePageFactory runDetailsPipelinePageFactory;
-
-    public Pipeline(String name) {
-        this(null, name);
-    }
-
-    public Pipeline(Folder folder, String name) {
-        if(folder == null) {
-            this.folder = Folder.folders();
-        } else {
-            this.folder = folder;
-        }
-        this.name = name;
-    }
+import com.google.inject.Inject;
+import com.google.inject.assistedinject.Assisted;
+import com.google.inject.assistedinject.AssistedInject;
+import io.blueocean.ath.api.classic.ClassicJobApi;
+import org.apache.commons.io.FileUtils;
 
-    public boolean isMultiBranch() {
-        return false;
-    }
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
 
-    public Folder getFolder() {
-        return folder;
+public class Pipeline extends AbstractPipeline {
```

Comment:
`ClassicPipeline` might be a better name, just to differentiate it from our more generic use of "pipeline" throughout.

Reply:
Ah yeah, i was trying to figure our job vs pipeline. Sounds lke a good idea